app/dts/behaviors/toggle_layer.dtsi

```diff
@@ -9,6 +9,7 @@
         /omit-if-no-ref/ tog: toggle_layer {
             compatible = "zmk,behavior-toggle-layer";
             #binding-cells = <1>;
+            friendly-name = "Layer Toggle";
```

Comment:
This is also typically called "Toggle Layer." Having a proper noun phrase rather than verb+action might be better in general, but I'd again tend to consistency.

Reply:
I'll defer to consistency for now, but I do think this reads better than "Toggle Layer", IMHO.